lib/license_finder/package_managers/yarn.rb

```diff
@@ -2,16 +2,20 @@
 
 module LicenseFinder
   class Yarn < PackageManager
-    SHELL_COMMAND = 'yarn licenses list --no-progress --json'
```

Comment:
is `--no-progress` a deprecated flag? or we don't need it anymore?

Reply:
or is that this flag is only in yarn 2 and not 1

Reply:
that flag is only supported in yarn 1, so I'm removing it from the base shell command and appending it only if it's yarn v1